components/base/sensorcontrolled/movestraight.go

```diff
@@ -54,6 +56,12 @@ func (sb *sensorBase) MoveStraight(
 		}
 	}
 
+	// if loop has been tuned but the values haven't been added to the config, error with tuned values
+	if (sb.configPIDVals[0].NeedsAutoTuning() && !(*sb.tunedVals)[0].NeedsAutoTuning()) ||
```

Comment:
do we want to error if tuning is in progress? I thought we did, since the apis won't actually work if the loop is still tuning